semgrep_output_v1.atd

```diff
@@ -106,7 +104,8 @@ type position
 }
 
 (* File path. less: could convert directly to Path class of pathlib library for Python *)
-type fpath <ocaml attr="deriving show"> = string wrap <ocaml module="ATDStringWrap.Fpath">
+type fpath <ocaml attr="deriving show"> =
```

Comment:
Is this change intentional? Looks like just whitespace

Reply:
yes just whitespace.